src/Core/Services/ApiService.cs

```diff
@@ -580,6 +580,15 @@ public Task<bool> GetKnownDeviceAsync(string email, string deviceIdentifier)
 
         #endregion
 
+        #region Configs
+
+        public async Task<ConfigResponse> GetConfigsAsync()
+        {
+            return await SendAsync<object, ConfigResponse>(HttpMethod.Get, $"/config/", null, false, true);
```

Comment:
⛏️ the `$` is not needed given that you're not passing any parameters to the string